compiler/analyzer.ml

```diff
@@ -53,6 +59,7 @@ let rec str_of_type = function
   | Void -> "Void"
   | List(l) -> sprintf "List[%s]" (str_of_type l)
   | Func(f) -> str_of_func f
+  | Dist(d) -> "Dist" (* Need to edit *)
```

Comment:
Yeah, just realized we need @dannyech to finish analyzer portion of this first! We'll wait for his changes before the merge.